presto-main/src/main/java/com/facebook/presto/translator/RowExpressionTranslatorVisitor.java

```diff
@@ -0,0 +1,90 @@
+package com.facebook.presto.translator;
+
+import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.spi.relation.CallExpression;
+import com.facebook.presto.spi.relation.ConstantExpression;
+import com.facebook.presto.spi.relation.InputReferenceExpression;
+import com.facebook.presto.spi.relation.LambdaDefinitionExpression;
+import com.facebook.presto.spi.relation.RowExpressionVisitor;
+import com.facebook.presto.spi.relation.SpecialFormExpression;
+import com.facebook.presto.spi.relation.VariableReferenceExpression;
+import com.facebook.presto.spi.translator.ExpressionTranslator;
+import com.facebook.presto.spi.translator.TargetExpression;
+
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+class RowExpressionTranslatorVisitor<T>
```

Comment:
Underlying adapter for RowExpression visitor. This calls functions on the ExpressionTranslator interface. This class allows to hide internals from the user (i.e. they don't have to care about the visitor, only have to implement functions that are pertinent to their use case)